examples/event/event_api_and_broker.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional
+
+from examples.event.event_aware_resource import HelloWorldExample
+from examples.event.event_broker import event_to_handler_map
+from lbz.events import EventBroker
+from lbz.exceptions import LambdaFWException
+from lbz.lambdas import LambdaSource
+from lbz.types import LambdaContext
+
+
+def handle(event: dict, context: LambdaContext) -> Optional[dict]:
+    try:
+        if LambdaSource.is_from(event, LambdaSource.API_GW):
+            return HelloWorldExample(event)().to_dict()
+        if LambdaSource.is_from(event, LambdaSource.EVENT_BRIDGE):
```

Comment:
```suggestion
        elif LambdaSource.is_from(event, LambdaSource.EVENT_BRIDGE):
```

Reply:
pylint disapproves - first if returns so el part of elif is redundant.

Reply:
Yeah, it makes sense 👍